package.json

```diff
@@ -48,11 +52,19 @@
     ]
   },
   "devDependencies": {
-    "ava": "^0.11.0",
-    "xo": "^0.12.1"
+    "ava": "^0.22.0",
+    "eslint": "^4.8.0",
+    "eslint-config-standard-with-typescript": "^1.0.1",
```

Comment:
This might be different than what you're used to. Preferences, you know. I had to pick something. So I picked my favorite. BTW, this is a package by me.